draft-ietf-acme-acme.md

```diff
@@ -840,6 +840,10 @@ externalAccountRequired (optional, boolean):
 new-account requests include an "externalAccountBinding" field associating the
 new account with an external account.
 
+certificateGET (optional, boolean):
+: If this field is present and set to "true", then the server allows GET
```

Comment:
the server, or the resource?

Reply:
The server.  This is supposed to apply to all certificate resources.